__tests__/server/routes/clusterRouter.test.ts

```diff
@@ -0,0 +1,52 @@
+import { connect, disconnect } from '../../../src/shared/models/mongoSetup';
+import { EndpointModel, Endpoint} from './../../../src/shared/models/endpointModel';
+import request from 'supertest';
+import app from './../../../src/server/index';
+
+describe('test out all our routes', () => {
+  jest.setTimeout(1 * 60 * 1000); //Sets a 60 second
+
+beforeAll(async () => {
+  await connect('mongodb+srv://admin:test@cluster0.dopf4.mongodb.net/DePaul?retryWrites=true&w=majority');
+  app.get('/', (req, res) => res.status(200).send([{ method: 'GET', endpoint: '/api/example' }, { method: 'POST', endpoint: '/api/example2' }]));
+  app.get('/load/1', (req, res) => res.status(200).send({ x: [1, 20, 12], y: [420, 300, 200]}));
+  app.get('/tree/1', (req, res) => res.status(200).send({name: 'Cluster1', children: [{name: 'get', children: [{name: '/api/login'}, {name:'/api/logout'}]}]}));
+  await EndpointModel.deleteMany();
+});
+
+afterEach(async () => {
+  await EndpointModel.deleteMany();
+});
+
+afterAll(async () => {
+  await disconnect();
+});
+
+test('Route: / || Middleware: getClusterList', async () => {
+  return request(app)
```

Comment:
You're not really testing the server, since you declared these routes here in the test lines 11-13